src/pages/data-globe/data-globe-initial-state.js

```diff
@@ -1,6 +1,8 @@
 export default {
   globe: {
-    isPaddingActive: true
+    isPaddingActive: true,
+    zoom: 3,
+    center: [16.9515536, 51.116959]
```

Comment:
why these values?

Reply:
There should be some default values, latitude and longitude, we can change it to anything else.